docs-src/content/getting-started/streaming.md

```diff
@@ -69,4 +69,37 @@ func main() {
 }
 ```
 {{% /tab %}}
+
+{{% tab "Python" %}}
+``` python
+import cubic
+
+serverAddress = '127.0.0.1:2727'
+
+client = cubic.Client(serverAddress)
+
+# use the first available model
+modelResp = client.ListModels()
+model = modelResp.models[0]
+
+cfg = cubic.RecognitionConfig(
+    model_id = model.Id
+)
+
+# audio file
+audio = open('test.raw', 'rb')
+
+# send streaming request to cubic and 
```

Comment:
How would this change if you want to stream audio from a mic and do live recognition? From this example I can't see what I'd change.

Reply:
I think we should document that client.StreamingRecognize takes any object that has the read() method. Perhaps have commented-out line or two that uses microphone input.